trim.production.js

```diff
@@ -71,23 +71,32 @@ function getDocument (trimId, callback) {
  * @param {Object[]}  container.records
  */
 
-
-/**
- * @param trimId
- * @param {containerCallback}  callback
- */
-function getContainer (trimId, callback) {
+function getContainerUsingMethod(method, trimId, securityToken, callback) {
   var options = {
-    url: url + '/GetContainer?trimid=' + trimId + '&securityToken=' + token,
+    url: url + '/' + method + '?trimid=' + trimId + '&securityToken=' + securityToken,
     json: true
   };
   debug('GET %s', options.url);
   request.get(options, function (err, res, responseBody) {
-    // responseBody has containerNo, subContainers, and records
     callback(err, responseBody);
   });
 }
 
+/**
+ * @param trimId
+ * @param {containerCallback}  callback
+ */
+function getContainer (trimId, callback) {
+  getContainerUsingMethod('GetContainer', trimId, token, callback);
+}
+
+/**
+ * @param trimId
+ * @param {containerCallback}  callback
+ */
+function getPrivateContainer (trimId, callback) {
+  getContainerUsingMethod('getPrivateContainer', trimId, token, callback);
```

Comment:
Id assume those are the literal trim methods. I'll check the docs tom gave us. The api is hand crafted by someone at mdc and they're pretty inconsistent so I'd assume they aren't following any conventions